src/desktop/components/clock/test/view.test.js

```diff
@@ -124,7 +124,7 @@ describe("ClockView", function () {
   })
 
   // TODO: components/clock/test/view.coffee
-  return xdescribe("#stateCallback", function () {
+  return describe.skip("#stateCallback", function () {
```

Comment:
wow ive never seen a `describe` on a return!

Reply:
me neither, I only fixed what jest complained, had to spend time in the other refactors 😅